src/views/accounts/RiskDetect/RiskHandlerFormatter/ReviewDrawer.vue

```diff
@@ -104,6 +104,18 @@ export default {
       } else {
         return this.selectedRows
       }
+    },
+    isBatch() {
+      // 如果是批量，则判断批量中所选的第一个的 Status，如果是在表格中，那么会存在 row.id 那么也应该出现 button
+      if (this.selectedRows.length > 0 && this.selectedRows[0] && this.selectedRows[0].status) {
+        return this.selectedRows[0].status.value === '0'
+      }
+
+      if (this.row.status) {
+        return this.row.status.value === '0'
+      }
+
+      return false
     }
   },
   methods: {
```

Comment:
The given code does not seem to be provided nor contains any regular programming languages like JavaScript, TypeScript etc. Hence I cannot perform any checks on it. Please provide more information if you intend to discuss or analyze a specific piece of code.